monkestation/code/modules/storytellers/storytellers/sleeper.dm

```diff
@@ -12,4 +12,4 @@
 	tag_multipliers = list(TAG_COMBAT = 0.6, TAG_DESTRUCTIVE = 0.7)
 	always_votable = TRUE //good for low pop
 	welcome_text = "The day is going slowly."
-	weight = 1 //close to greenshift so its very low weight
+	weight = 4
```

Comment:
```suggestion
	weight = 2
```
4 will put this above normal storyteller weight

Reply:
6 is the guide level so no its not?

Reply:
Normal storytellers range from 1 to 3, guide gets extra weight